lib/broadside/configuration.rb

```diff
@@ -25,15 +25,17 @@ class Configuration
     validates_each(:aws) do |record, _, val|
       [:region, :credentials].each { |v| record.errors.add("aws.#{v}") unless val.public_send(v) }
     end
+    validates_each :ssh, allow_nil: true do |record, attr, val|
+      record.errors.add(attr, 'is not a hash') unless val.is_a?(Hash)
+      record.errors.add(attr, 'must contain a user') unless val[:user]
```

Comment:
good point. i'll address the `@` with https://github.com/lumoslabs/broadside/issues/11